cmd/main.go

```diff
@@ -1,45 +1,47 @@
 package main
 
 import (
-	"log"
 	"os"
+	"time"
+
+	"github.com/rs/zerolog"
 
 	"github.com/unagex/kondense/pkg/controller"
 	"github.com/unagex/kondense/pkg/utils"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
 func main() {
-	// create logger.
-	l := log.Default()
-	l.SetFlags(log.Lshortfile | log.LstdFlags)
+	l := zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: time.RFC3339}).Level(zerolog.TraceLevel).With().Timestamp().Logger()
```

Comment:
I think we can use instead `"github.com/rs/zerolog/log"` (preconfigured zerolog) first to reduce code complexity and because we don't need to use special zerolog features yet.

Reply:
as we need to initialize a logger (even the default one), since we must pass a pointer when creating the Reconciler, 
Why not use this line to configure the logger propertly ?

Reply:
We can remove the field in the struct entirely, we would not need to pass the pointer anymore.